packages/common/container-definitions/src/deltas.ts

```diff
@@ -241,6 +227,25 @@ export interface IDeltaManager<T, U>
 	submitSignal(content: any, targetClientId?: string): void;
 }
 
+/**
+ * DeltaManager which is used internally by the Fluid layers and not exposed to the end users.
+ * @sealed
+ * @legacy
+ * @alpha
```

Comment:
If it's only used internally shouldn't this be `@internal`?

Reply:
Oh, I see the new type is used in an existing `@alpha` API `toDeltaManagerInternal()`. But then I don't quite get the point of this change. We're removing the queues from one alpha interface... to put them in another one where they are not marked as deprecated, which seems wrong?

Reply:
Fixed.